src/internal_modules/roc_netio/target_libuv/roc_netio/udp_receiver_port.cpp

```diff
@@ -286,6 +286,7 @@ void UdpReceiverPort::recv_cb_(uv_udp_t* handle,
 
     pp->udp()->src_addr = src_addr;
     pp->udp()->dst_addr = self.config_.bind_address;
+    pp->udp()->recieve_timestamp = core::timestamp(core::ClockMonotonic);
```

Comment:
For timestamping audio frames we use `core::ClockUnix`

cc @gavv

Reply:
Indeed, I think recieve_timestamp should use the same.